contracts/Makefile

```diff
@@ -2,8 +2,12 @@ deps:
 	yarn
 
 compile: deps
-	# compile.sh compiles the contracts and creates go bindings
-	./compile.sh
+	forge build
+
+# make bindings compiles the contracts and creates go bindings
+bindings: compile
```

Comment:
nit - is the compile step cached at all or is it re-ran every-time `make bindings` is called?